src/main/java/de/bottlecaps/railroad/core/FontWidth.html

```diff
@@ -1,6 +1,9 @@
 <html>
     <head>
         <meta http-equiv="content-type" content="text/html; charset=utf-8"/>
+<style>
+ @import url('https://fonts.googleapis.com/css2?family=Encode+Sans+Expanded:wght@100;300&family=Fira+Code&family=Inter:wght@300;400;500;700&display=swap');
```

Comment:
Did you mean to include encode sans and fira code here?

Reply:
ah, I just copy pasted our CSS rule from the main repo